src/components/PageThemes.jsx

```diff
@@ -0,0 +1,21 @@
+import { useOutletContext } from "react-router-dom";
+import Button from "./Button"
+
+const HomePage = () => {
+  const { setThemeColor } = useOutletContext();
```

Comment:
Interesting that you used this `hook`. Any reason to why you did it this way instead how we did it in class?